crates/holochain_conductor_api/src/config/conductor.rs

```diff
@@ -358,9 +360,9 @@ mod tests {
       type: lair_server_in_proc
 
     dpki:
-      dna_path: path/to/dna.dna
-      network_seed: "deepkey-main"
-      device_seed_lair_tag: "device-seed"
+      dna_path: ~
```

Comment:
Likewise with this test please, can we have the one with and without the feature? Then it's easier to remove the feature later and we also know we're still testing the same level of functionality we were before adding this feature